base/exports.jl

```diff
@@ -1197,6 +1197,9 @@ export
     init_worker,
     interrupt,
     isready,
+    IC_MASTER_SLAVE,
+    IC_ALL_TO_ALL,
+    IC_CUSTOM,
```

Comment:
Is it worth/necessary exporting three new names to have this new option? They are only used in `set_interconnect`, rught? So maybe just have `set_interconnect(s::Symbol)` with `:master_slave`, `:all_to_all` and `:custom` and then have the enums unexported. I'm also wondering if it is enough to call the function `interconnect` (or `topology`) and avoid the underscore.